.github/workflows/ci-integration.yml

```diff
@@ -38,13 +44,13 @@ jobs:
           #   ccm_version: 6.7.17
           #   is_dse: true
           - cassandra_version: dse-6.8
-            ccm_version: 6.8.61
+            ccm_version: 6.8.24
             is_dse: true
 
     steps:
-      - uses: actions/checkout@v6
```

Comment:
These GHA versions shouldn't cause any issues FWIW and should be kept at the latest if they don't cause any known problems.